SecurityModel.md

```diff
@@ -0,0 +1,42 @@
+Information on how accounts and permissions are handled within Wubloader.
+
+### Google Authentication
+
+Google OAuth is used to authenticate users and return a user token. The Token is then passed alongside calls that need authentication 
+and is validated server-side against a Google API. The API returns the authenticated user's email address (along with other basic 
+profile information).
+
+The email address is then used to look up the user in the database to check their permissions.
+
+The current plan is to only authenticate/authorize users on datebase updates coming from Thrimbletrimmer; and since most use cases will 
+only have a single update event every few minutes, we can authenticate only as need - rather than authenticating on page load and storing session IDs.
+
+### Wubloader Authorization
+
+Current tasks via Thrimshim are:
+* `get_row`: Return a single row from the database by ID. Unauthenticated.
+* `update_row`: Updates row in the Events table. Authenticated and limited to Editors and Admins (who can update different sets of rows).
```

Comment:
the existing thrimshim update_row call is basically the `submit_edits` call mentioned below, it can't do anything else.